### DISCUSSION TITLE
"FolderNode" to group nodes

### DISCUSSION THREAD

**Discussion by venilark** [godotengine/godot-proposals#7240] [Engine Core]
If we want to group 3D nodes we must put them as children of a parent 3D node, which means more nested operations and maybe extra to_global/to_local operations the user must do.
With a FolderNode we could put everything we want inside it and it simply acts as a dropdown with the show/hide arrow.

Right now we have:
-Level
  -Coins
   -CoinsArea1
      -CoinsSubarea1
        -Coin1
        -Coin2
   -CoinsArea2
   -...

Coin1 must inherit the transform of CoinsSubarea1, that one from CoinsArea1, then Coins and finally Level
With folders the hierarchy will be the same but Coin1 would actually be the direct child of Level

**Comment by paddy-exe** [comment]
If I understand your problem correctly, you can use the basic Node for this just as well and just rename it however you like it. It doesn't have a transform so it won't manipulate transforms of its children
